blues/buildmol.py

```diff
@@ -0,0 +1,89 @@
+import os
+from IPython.display import Image
+import numpy as np
+from rdkit import Chem
+from rdkit.Chem import AllChem, Draw
+
+
+def order_atoms(atom_file):
+    """
+    Enumerates the .txt file containing all atoms in the molecule, order the same as JANPA output and eliminate whitespace
```

Comment:
Limit line length to 72 characters for docstrings, also add a blank line between function description and `Parameters` section